src/features/currentTodo.ts

```diff
@@ -29,6 +29,12 @@ const currentTodoReducer = (
 ): State => {
   switch (action.type) {
     // Implement all actions here
+    case 'currentTodo/SET':
```

Comment:
you can put those values inside enum as well

Reply:
done